ppafm/cpp/Forces.h

```diff
@@ -203,6 +204,28 @@ inline double invR8_func(double r2,double &df){
     return            invR8;
 }
 
+// DFT-D3 force between two atoms
+inline double addAtomDFTD3(const Vec3d& dR, Vec3d& fout, double c6, double c8, double R0_6, double R0_8) {
+
+    double r2 = dR.norm2();
```

Comment:
maybe I would add some tiny safety value (like `double r2 = dR.norm2() + 1.e-8;`) just in case when the two atoms accidently overlap. But perhaps this does not solve much.

Reply:
It shouldn't be required here. Notice that `r2` (or the higher powers) are not used directly as the divisor here, but rather the damping constants `R0_6` and `R0_8` are added to it before the division. And the damping constants should never be zero.

Reply:
yes, you are right, I did not look carefylly